src/interactive-window/debugger/startupCodeProvider.web.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { inject, injectable } from 'inversify';
+import { Uri } from 'vscode';
+import { getAssociatedNotebookDocument } from '../../kernels/helpers';
+import { IKernel, IStartupCodeProvider, StartupCodePriority } from '../../kernels/types';
+import { InteractiveWindowView } from '../../platform/common/constants';
+import { IFileSystem } from '../../platform/common/platform/types';
+import { IExtensionContext } from '../../platform/common/types';
+
+@injectable()
+export class InteractiveWindowDebuggingStartupCodeProvider implements IStartupCodeProvider {
```

Comment:
Seems like these could be the same. Just use the fs.readFile on the extension URI for both.

Reply:
From what I remember, this same code applies to desktop extension as well, when debugging using remote jupyter kernels. Hence we need this same code for non-web extension as well

Reply:
aaah, I can see that this is the same code in `InteractiveWindowDebuggingStartupCodeProvider.node & .web`
Yeah, would prefer if we could have the same class to do both, rather than having two seprate classes with the same logic, then the logic for debugging remote jupyter kernels is in one place, instead of being split across two files.

Reply:
yep, agree.